pkg/golinters/gofuncor/testdata/gofuncor.go

```diff
@@ -0,0 +1,39 @@
+package testdata
```

Comment:
Tests won't work without this line:
```suggestion
//golangcitest:args -Egofuncor
package testdata
```

Reply:
line added and now the integration test works